tests/tools/oeapkman/zlib/host/host.c

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/host.h>
+#include <openenclave/internal/error.h>
+#include <openenclave/internal/tests.h>
+#include <stdio.h>
+#include "test_u.h"
+
+int main(int argc, char* argv[])
+{
+    oe_result_t result;
+    oe_enclave_t* enclave = NULL;
+    int return_val;
+    if (argc < 2)
+    {
+        fprintf(stderr, "Usage: %s ENCLAVE_PATH [-d]\n", argv[0]);
+        return 1;
+    }
+
+    const uint32_t flags = oe_get_create_flags();
+
+    if ((result = oe_create_test_enclave(
+             argv[1], OE_ENCLAVE_TYPE_SGX, flags, NULL, 0, &enclave)) != OE_OK)
```

Comment:
why SGX instead of AUTO?  I don't see any SGX-only code in this file and tests/tools/oeapkman/zlib/test.edl explicitly allows optee too, so this looks like a bug.

Reply:
See explanation above.

Reply:
Which explanation is referred to?   I saw a comment saying optee support was added.